ci_framework/roles/libvirt_manager/molecule/deploy_layout/converge.yml

```diff
@@ -218,6 +220,71 @@
                 that:
                   - cmp_1.host.ip == '192.168.140.101'
 
+    - name: Verify existence of storage pool.
+      ansible.builtin.command:
+        cmd: "virsh -c qemu:///system pool-info cifmw-pool"
+
+    - name: Volumes testing
+      block:
+        - name: Gather the list of volumes
+          register: vol_list
+          ansible.builtin.shell:
+            cmd: |
+              set -o pipefail
+              virsh -c qemu:///system vol-list --pool cifmw-pool | \
+              grep -c cifmw-compute
+
+        - name: The list of gathered volumes
+          ansible.builtin.debug:
+            var: vol_list.stdout_lines
+
+        - name: Verify the number of volumes created
+          vars:
+            expect_vol_num: >-
+              {{
+                (cifmw_libvirt_manager_configuration.vms.compute.amount | int) *
+                (cifmw_libvirt_manager_configuration.vms.compute.extra_disks_num | int)
+              }}
+            found_vol_num: "{{ vol_list.stdout | int }}"
+          ansible.builtin.assert:
+            that:
+              - found_vol_num == expect_vol_num
+
+    - name: Volume attachment testing
+      block:
+        - name: Gather the compute domain names.
+          register: compute_list
+          ansible.builtin.shell:
+            cmd: |
+              set -o pipefail
+              virsh -c qemu:///system list --name | grep compute
+
+        - name: Gather the volumes attached
+          register: vols_attach
+          ansible.builtin.shell:
+            cmd: |
```

Comment:
maybe the dump_xml + xml module may be smarter? There are some examples already in this same file, related to network interfaces.

Reply:
Okay.